### ISSUE TITLE
Second Column in CSV Needs to be Polygon ID

### ISSUE THREAD

**Issue by davodogster** [charlotte-pel/temporalCNN#1]
Hi Charlotte

Love the repository, and the work that you and your team do. The land-cover map of Victoria is very impressive.

In your other TempCNN Repo you state that the second column in the CSV needs to be PolygonID. It appears that the same is required for the model in this Repo. 

Also, are you able to upload a prediction script which generates the same architecture that was trained on, then loads the weights from the results folder and predicts on a CSV and stores the class prediction results in CSV or numpy format? 
The ultimate goal is to reshape this prediction, and then georeference it and visualize the predictions.

Cheers,
Sam

**Comment by fpetitjean** [comment]
Hi Sam, 
Charlotte is currently attending MultiTemp but is preparing these for you. 
Cheers, 
François

**Comment by charlotte-pel** [comment]
Dear Sam,

Thank you for the interest in our projects, and thank you for pointing out the issue with PolygonID. I have amended the README.md file.

Regarding your second request, I have created a write_output.py script that takes a csv file or a tiff image as an input, and that outputs the predictions in a csv file or a tiff image. Let us know if it works or if you need more help.

Cheers,
Charlotte.

**Comment by davodogster** [comment]
Thank you for doing that Charlotte. How do I store the minMax values when training?

main(args.model_path, args.test_file, args.result_file, args.proba, args.feature)
  File "write_output.py", line 37, in main
    assert False, "ERR: min-max values needs to be stored during training"
AssertionError: ERR: min-max values needs to be stored during training

Sam

**Comment by charlotte-pel** [comment]
Dear Sam,

When creating the `write_output.py` script , I did a slight modification to `run_archi.py` (and also `run_main_archi.py`) to automatically save the min-max values in the same folder than the model.

In your case, that means you only need to relaunch `run_archi.py` (with the same parameters) to automatically save min-max values computed on the training dataset in a csv file. The model will not be trained again if `res_file` (`resultOA_XXX.csv`) has been corrected save.

I hope it does help.

Cheers, Charlotte.
